grype/matcher/rpmdb/matcher.go

```diff
@@ -19,15 +21,15 @@ var rpmPackageNamePattern = regexp.MustCompile(`(?P<name>^[a-zA-Z0-9\-]+)-\d+\.`
 type Matcher struct {
 }
 
-func (m *Matcher) PackageTypes() []pkg.Type {
-	return []pkg.Type{pkg.RpmPkg}
+func (m *Matcher) PackageTypes() []syftPkg.Type {
+	return []syftPkg.Type{syftPkg.RpmPkg}
```

Comment:
(for later) It might be nice to have a `ToSlice` kind of helper method on Syft's `Pkg.Type` type